### DISCUSSION TITLE
ios shortcut set

### DISCUSSION THREAD

**Discussion by mareczek82** [LeoKlaus/plappa#227] [Q&A]
Hi, 

I just found this amazing app. Mostly i use it with carplay, but sometimes i use it via BT. 

Could someone help me how to set iphone to lunch Plappa when phone connects with my car BT? I set shortcut but i need to click on screen to lunch app, i would like to work it that i idnt need to take phone out of my pocket.

**Comment by LeoKlaus** [comment]
I set up the following automation in the Shortcuts app:
![image](https://github.com/user-attachments/assets/ce56d216-aac9-4f82-a8b4-373bf7ac1bbb)

**Reply by mareczek82** [reply]
OMG, I was blind. 

Thank you!
